### PR TITLE
Fixes #21750: versions in Rudder doc are invalid

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/21750

Replacing previous PR: https://github.com/Normation/rudder-doc/pull/924

### CODE REVIEW

**Comment by ncharles** [comment]
PR updated with a new commit

File: src/reference/modules/usage/pages/variables.adoc [hunk 1]
```diff
@@ -633,7 +631,7 @@ The resulting properties will be:
 
 Agents can define variables:
 
-* For the Linux agent, https://docs.cfengine.com/docs/3.15/reference-special-variables-sys.html[CFEngine system variable] are available.
+* For the classic (Unix) agent, https://docs.cfengine.com/docs/3.18/reference-special-variables-sys.html[CFEngine system variable] are available.
```
Comment (fanf) [inline src/reference/modules/usage/pages/variables.adoc:634]: I think we chose to use the term "Linux" agent when we chose to use "Windows" (for DSC one)

Comment (amousset) [inline src/reference/modules/usage/pages/variables.adoc:634]: We renamed "classic"  to "linux" in 7.2 https://rudderio.slack.com/archives/G97HW5U20/p1656330279100489?thread_ts=1656315436.355709&cid=G97HW5U20

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**